airflow/providers/openlineage/provider.yaml

```diff
@@ -83,7 +83,8 @@ config:
           Exclude some Operators from emitting OpenLineage events by passing a string of semicolon separated
           full import paths of Operators to disable.
         type: string
-        example: "airflow.operators.bash.BashOperator;airflow.operators.python.PythonOperator"
+        example: "airflow.providers.standard.operators.bash.BashOperator;
```

Comment:
I am not sure about this change impact, will it cause any issues? when this provider release in the next provider release cycle, without standard provider? could you please give your thoughts?

Reply:
@eladkal This in-deed is a chicken-and-egg problem we had last time with FAB as well. How about if we make a "small" PR such that the standard provoder is ready, release this once (manually) to have it as an empty or half-bake version such that we can un-block this loop?

Reply:
Thanks jens.

Yes small release will help us break the loop, so there's no need to skip the k8s and Docker tests that were recently marked as skipped in the latest commit.

Additionally, hooks related to FileHook and PackageIndexHook are being used in the provider manager, which could also cause test failures when moved to the standard provider.

Reply:
Hi @eladkal any thoughts on this please?

Reply:
Why do we need a release? We can add it to the standard list well before we do a release of providers.

Reply:
sorry i could have referenced code links:

https://github.com/apache/airflow/blob/main/airflow/providers_manager.py#L39
https://github.com/apache/airflow/blob/main/airflow/providers_manager.py#L40

Reply:
> Why do we need a release? We can add it to the standard list well before we do a release of providers.

@ashb i have tried adding standard provider to prod_image_installed_providers.txt , but this package is not getting installed in container image, which cause the test failures in here `kubernetes_tests/test_kubernetes_executor.py` and `docker_tests/test_docker_compose_quick_start.py`. 

celery executor utils is using bash/python operator imports:

https://github.com/apache/airflow/pull/42252/files#diff-0e0d141920152b568dd569500b73ebaca58893fa2a98255ef5a565743f5e374aR119

As a work around i have added skip check, i feel this is not good idea to skip these tests , may be you can have more view on these. 

https://github.com/apache/airflow/pull/42252/commits/bd03162e007be7268b0ebb7b650887ea5f9141fc#diff-8866ef4555c16e5398e9e5f7194c3b3251c69e2adba9b57bc899d421305d0b17R31

https://github.com/apache/airflow/pull/42252/commits/b553358ead17b768c03ec1c3a895a6b4520a0aeb#diff-0f21d6ea329e8ba2276c0c2c945d7325ac64e6dae24f27a858c7cf0fab6760e5R70

Reply:
> > Additionally, hooks related to FileHook and PackageIndexHook are being used in the provider manager, which could also cause test failures when moved to the standard provider
> 
> @jscheffl You added this -- looking at the code (but not the PR desc, forgive me if it's mentioned there) why does the provider manager need to import these two classes eagerly the way it does?

@ashb ... don't know... did I add the imports? I don't know the reason or we never had the problem. If we make the import lazy and this would fix the problem I am okay. But also providers manager is loading a lot of stuff during start, am not sure whether this resolves or just makes the problems a step later. It is worth trying to lazy import. Might even improve startup time as side effect if we are lucky :-D

Reply:
See the devlist message that I sent https://lists.apache.org/thread/1gshy5cjmp9wz5v8dozyh64jj3dyn5s4 - you cannot add standard provider to "prod_image_installed_providers.txt" because it's not released in PyPI (yet) - but we can release it's pre-release version manually.

Reply:
FYI: We do not need to switch "not-read" temporarily for that - we can use direct breeze command to build the package with .dev0 version and upload it to PyPI. I think that's the easiest way.

This is as simple as:

```bash
breeze release-management prepare-provider-packages standard --version-suffix-for-pypi dev0
```

Followed by:

```bash
twine upload dist/apache_airflow_providers_standard-1.0.0.dev0-py3-none-any.whl
```

The only prerequisite is that person doing it has PyPI credentials configured.

This way we can upload it any time without waiting for regular release process - i.e. when we move some stuff to standard provider, we can upload it to PyPI from the PR branch, before we merge the PR.

Reply:
Thank you for this, I will remove the skipif checks added to k8s and docker tests on this pr.

Reply:
> I will remove the skipif checks added to k8s and docker tests on this pr.

Sure, We will need to release the "dev0" version of providers first after we get to consensus that this is the right thing to do.

Reply:
OK. I uploaded the standard package .dev0 to try it - https://pypi.org/project/apache-airflow-providers-standard/1.0.0.dev0/ -> you can try to remove the skip-if in your PR now @gopidesupavan and we will take it from there.

Reply:
Thank you :) updated now.

Reply:
@potiuk it looks like not adding to container image https://github.com/apache/airflow/actions/runs/11159243493/job/31018063584?pr=42252#step:12:606 , any constrain file changes required? 

failure here: https://github.com/apache/airflow/actions/runs/11159243533/job/31018299717?pr=42252#step:8:1284

Reply:
Might it be related to https://pip.pypa.io/en/stable/cli/pip_install/#pre-release-versions and we need to pin specifically to the dev version to have it installed?

Reply:
I am not 100% sure and we never had this use case before but I assume it must be changed in hatch_build.py:52 and there the currect dev version must be pinned. (Like the version of FAB also is in line 42)

Reply:
Thanks jens, updated let me check if this picks up or not.

Reply:
it didnt installed. trying another approach, gone through the prod-image-build file, there are steps installing the packages, added extra step to install standard provider, let me check if this installs package or not.

Reply:
No luck, even updated CHICKEN_EGG_PROVIDERS in global constants, build is not considering this.

Reply:
Mhm... no idea... @potiuk I assume we need your master expert expertise :-D

Reply:
Note that the failed backcompat in the last build is (hopefully) fixed in https://github.com/apache/airflow/pull/42750 .. the other might be an in-stability... maybe we need to re-base once the main is green as well.

Reply:
Woohooo finally all the tests went through green , except one but this is not related to this change. 😄 

Thanks all for the help....